services/movie_session.py

```diff
@@ -0,0 +1,54 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall, Movie, MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
```

Comment:
The `create_movie_session` function uses `get` to retrieve `Movie` and `CinemaHall` instances, which will raise a `DoesNotExist` exception if the IDs are not found. Consider handling these exceptions to provide a more user-friendly error message or behavior.